components/organisms/Drawer.js

```diff
@@ -0,0 +1,74 @@
+import PropTypes from "prop-types";
+import React, { useState } from "react";
```

Comment:
`useState` can be removed if it's not being used.

Reply:
Ok I'm on it.